src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -192,6 +193,11 @@ public function iConfirmMyOrder(): void
                 'notes' => $notes,
             ], \JSON_THROW_ON_ERROR)
         );
+
+        $this->sharedStorage->set(
+            'order_number',
+            $this->responseChecker->getValue($this->client->getResponse(), 'number')
```

Comment:
Is the number not `null` here? I guess that it should be because the order number is assigned after checkout

Reply:
The number is already available here:
<img width="1014" alt="Screenshot 2020-08-04 at 14 44 26" src="https://user-images.githubusercontent.com/39232096/89295488-600f2b00-d661-11ea-962c-795c4d60ba37.png">